integration/client/container_linux_test.go

```diff
@@ -1453,7 +1453,7 @@ func TestIssue9103(t *testing.T) {
 			desc: "should be created status",
 			cntrOpts: []NewContainerOpts{
 				WithNewSpec(oci.WithImageConfig(image),
-					withProcessArgs("sleep", "30"),
+					withProcessArgs("sleep", "inf"),
```

Comment:
Yes. The test case just calls `NewTask` so that the runc-init is still waiting for signal from `task.Start`.
When the `handleProcessExit` doesn't have chance to update the status, the `task.Status` will return `Created` status. So I think we can use `task.Wait` to ensure that we can get stopped status when the expectedStatus is stopped.

Reply:
Thanks, I missed this.